modules/tooltip/main/index.coffee

```diff
@@ -0,0 +1,21 @@
+
+hx.tooltip = (selector, opts) ->
+
+  options = hx.merge({
+    message: ''
+    align: 'right'
+  }, opts)
+
+  content = (element) ->
+    hx.select(element)
+      .text(options.message)
+      .classed('hx-tooltip', true)
+
+  dropdownOptions = {
+    mode: 'hover',
+    align: options.align
+  }
+
+  new hx.Dropdown(hx.select(selector).node(), content, dropdownOptions)
```

Comment:
This could potentially be an extension to dropdown as an option that adds the 'hx-tooltip' class for you when passing in a string as the `dropdownContent` - It doesn't feel like it's _different_ enough from a dropdown to be called a tooltip...